actors/builtin/miner/miner_actor.go

```diff
@@ -1504,19 +1516,6 @@ func requestTerminateDeals(rt Runtime, dealIDs []abi.DealID) {
 	builtin.RequireSuccess(rt, code, "failed to terminate deals %v, exit code %v", dealIDs, code)
 }
 
-func requestTerminateAllDeals(rt Runtime, st *State) {
```

Comment:
Oh, I just used this to terminate deals on consensus fault (#472). It would be great if you could leave it. Otherwise one of us will need to deal with it rebasing.